polaris.yml

```diff
@@ -0,0 +1,27 @@
+version: "1"
+project:
+  name: ${scm.git.repo}
+  branch: ${env.SCM_BRANCH}
+  revision:
+    name: ${scm.git.commit}
+    date: ${scm.git.commit.date}
+  properties:
+    GROUP: ${env.POLARIS_GROUP_ID}
+    PSC_ID: ${env.POLARIS_PSC_ID}
+  groups:
+    1055Visma.net AutoPay: Administrator

```

Comment:
Could you check if it's possible to use environment variables here, for both key and value? (set them in the Jenkins job)

Intention is not to reveal these "internal" values in publicly available source code.